севрюгин home work 4

FishSev opened the pull request:
Выполнил первую часть домашки

FishSev committed:
выполняю задание 4, часть 1
- в папке part1 png-картинки с соответсвующими частями задания
- уверен ли я, что всё сделал правильно? Да (на 99%) :)

FishSev committed:
подправил типы связей

Tazya reviewed (commented):
Задание с планетами.
Расстояние до планеты нельзя привязать к самой планете, т.к. это не её характеристика, оно зависит от пункта отправления

Задание с кинотеатрами.
Некритично, но наверное добавил бы ещё сущность зала, для обеспечения расширяемости функционала.

FishSev committed:
исправление замечаний
1) Космопорт. Согласен с замечанием с точки зрения последующего
   расширения бизнеса и увеличения количества космопортов. Просто в
   задании был один космопорт (условно на Земле) и от него расстояние до
   планеты определяется этой планетой.
2) Кинотеатр. Добавил сущность для расширения функционала в последующем.

FishSev commented:
Спасибо за замечания. В коммите ответы по замечаниям расписал.

Tazya reviewed (approved):
Approved

FishSev committed:
выполняю вторую часть ДЗ 4

FishSev commented:
Добавил вторую часть домашки в ту же ветку

Tazya reviewed (approved):
Approved. Всё отлично

FishSev commented:
задание выполнено
